packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -216,6 +216,16 @@ export type AutocompleteProps<
    */
   getIsOptionEqualToValue?: (option: OptionType, value: OptionType) => boolean;
 
+  /**
+   * Used to customize the filtering logic for the options in the dropdown
+   */
+  getOptions?: MuiAutocompleteProps<
```

Comment:
We should probably just name this `filterOptions`. Makes more sense that way I think. `getOptions` makes me think this has to do with async option loading. Any thoughts?

Reply:
I would think `getOptions` means it's a callback function to get your options. Is this an object or a function?

Reply:
I guess `filterOptions` could also be a function name: "filter the options".

Depends on the use case.